### PR TITLE
Add mintso config parameters

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/18997
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -264,6 +264,15 @@ delta_index_cache_size = 0
...
     enable_resource_control = true
 
+    ## New in v6.0.0. This item is used for [MinTSO scheduler](/tiflash/tiflash-mintso-scheduler.md) and specifies the max number of threads that one resource group could use. The default value is 5000.
+    task_scheduler_thread_soft_limit = 5000
+
+    ## New in v6.0.0. This item is used for [MinTSO scheduler](/tiflash/tiflash-mintso-scheduler.md) and specifies the max number of threads in global scope. The default value is 10000.
+    task_scheduler_thread_hard_limit = 10000
+
+    ## New in v6.4.0. This item is used for [MinTSO scheduler](/tiflash/tiflash-mintso-scheduler.md) and specifies the max number of running queries in a TiFlash instance. The default value is 0 which means two times the vcpu.
+    task_scheduler_active_set_soft_limit = 0
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:274]:
```suggestion
    ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_soft_limit = 5000

    ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000.
    task_scheduler_thread_hard_limit = 10000

    ## New in v6.4.0. This item is used for MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs.
    task_scheduler_active_set_soft_limit = 0
```

Comment (xzhangxian1008) [reply tiflash/tiflash-configuration.md]: > ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
>     task_scheduler_thread_soft_limit = 5000
>     ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000.
>     task_scheduler_thread_hard_limit = 10000
>     ## New in v6.4.0. This item is used for MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs.
>     task_scheduler_active_set_soft_limit = 0

done

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -264,6 +264,15 @@ delta_index_cache_size = 0
...
     enable_resource_control = true
 
+    ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
+    task_scheduler_thread_soft_limit = 5000
+
+    ## New in v6.0.0. This item is used for MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
+    task_scheduler_thread_hard_limit = 10000
+
+    ## New in v6.4.0. This item is used for MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
+    task_scheduler_active_set_soft_limit = 0
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:274]:
```suggestion
    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_soft_limit = 5000

    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_hard_limit = 10000

    ## New in v6.4.0. This item is used for the MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_active_set_soft_limit = 0
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -264,6 +264,15 @@ delta_index_cache_size = 0
...
     enable_resource_control = true
 
+    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
+    task_scheduler_thread_soft_limit = 5000
+
+    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
+    task_scheduler_thread_hard_limit = 10000
+
+    ## New in v6.4.0. This item is used for the MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
+    task_scheduler_active_set_soft_limit = 0
```
Comment (lilin90) [inline tiflash/tiflash-configuration.md:274]:
```suggestion
    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads that one resource group can use. The default value is 5000. For details about the MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_soft_limit = 5000

    ## New in v6.0.0. This item is used for the MinTSO scheduler. It specifies the maximum number of threads in the global scope. The default value is 10000. For details about the MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_thread_hard_limit = 10000

    ## New in v6.4.0. This item is used for the MinTSO scheduler. It specifies the maximum number of queries that can run simultaneously in a TiFlash instance. The default value is 0, which means twice the number of vCPUs. For details about the MinTSO scheduler, see https://docs.pingcap.com/tidb/dev/tiflash-mintso-scheduler.
    task_scheduler_active_set_soft_limit = 0
```

**Review by windtalker — APPROVED** [review]
lgtm

**Pull request merged**